lib/ui/src/core/shortcuts.js

```diff
@@ -31,7 +32,7 @@ export default function initShortcuts({ store }) {
   const api = {
     // Getting and setting shortcuts
     getShortcutKeys() {
-      return store.getState().shortcuts;
+      return store.getState().shortcuts || getInitialState().shortcuts;
```

Comment:
@ndelangen do you remember why this line is necessary? I thought we put the initial/default state into the store right away on line 225 below?

Reply:
Might be unnecessary now, let me check..